authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+# Process to publish GitHub Actions for Azure
```

Comment:
Should the title be "Process of Authoring GitHub Actions for Azure"? For publishing, author has to go through different bars.

Reply:
Done. Renamed the doc.